sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -60,6 +60,16 @@ static const bool UseCopyEngineForD2DCopy = [] {
   return (CopyEngineForD2DCopy && (std::stoi(CopyEngineForD2DCopy) != 0));
 }();
 
+// This is an experiemental option that allows the use of copy engine, if
+// available in the device, in Level Zero plugin for copy operations submitted
+// to an in-order queue. The default is 0.
+static const bool UseCopyEngineForInOrderQueue = [] {
```

Comment:
I do not see any uses of it, why?

Reply:
@asudarsa  @smaslov-intel  Use is below. Arvind, make sure you did not miss it.
Old
===
  bool UseCopyEngine = PreferCopyEngine && Queue->Device->hasCopyEngine();

new 
==== 
  bool UseCopyEngine = (!(Queue->isInOrderQueue()) ||
                           UseCopyEngineForInOrderQueue) && PreferCopyEngine &&
                          Queue->Device->hasCopyEngine();